plugins/scaffolder/package.json

```diff
@@ -38,7 +38,7 @@
   "scripts": {
     "build": "backstage-cli package build",
     "start": "backstage-cli package start",
-    "lint": "backstage-cli package lint",
+    "lint": "backsage-cli package lint",
```

Comment:
This looks like a backspace/delete key pressed on the wrong window.

Reply:
Fixed 👍